src/main/java/com/mindera/pizza/domain/ingredient/Ingredient.java

```diff
@@ -31,6 +33,7 @@ public class Ingredient extends DatabaseTimestamps{
 
     protected Ingredient() {}
 
+    @Builder
```

Comment:
esta anotação não devia ficar associado à class?

Reply:
e assim não precisas de fazer o this.name=name, ...

Reply:
e qual o problema disso? estás a expor uma ação que vai ser usada por ti, só não tens de chamar no builder as propriedades que não queres.
neste caso com este construtor podes até criar um objecto sem definir o id mas depois podes fazer o set do id, indiretamente essa propriedade fica exposta também